event-publisher/src/main/java/uk/gov/ons/ctp/common/event/model/CollectionExercise.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.ons.ctp.common.event.model;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import java.util.Date;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.ons.ctp.common.jackson.CustomDateSerialiser;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class CollectionExercise implements EventPayload {
+  private String collectionExerciseId;
+  private String surveyId;
+  private String name;
+  private String reference;
```

Comment:
where is the requirement for `reference` ?

Reply:
https://github.com/ONSdigital/ssdc-shared-events/blob/main/event_dictionary/v0.3_RELEASE/collectionExerciseUpdate.schema.json

I got the event definition from here, it contains `reference` and the fields mentioned in the comment below

Reply:
OK , it has changed from the DRAFT to the RELEASE